SourceryTests/Models/MethodSpec.swift

```diff
@@ -58,6 +58,12 @@ class MethodSpec: QuickSpec {
                 expect(Method(name: "foo()", definedInTypeName: TypeName("BarAlias", actualTypeName: TypeName("Bar"))).actualDefinedInTypeName).to(equal(TypeName("Bar")))
             }
 
+            it("reports isDeinitializer properly") {
+                expect(sut?.isDeinitializer).to(beFalse())
+                expect(Method(name: "deinitObects() {}").isDeinitializer).to(beFalse())
```

Comment:
Add the missing `j` here for `deinitObjects` 😉

Reply:
Ah, silly me, this is fixed in 24f91aa 😄